src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/CommunityAPIService.kt

```diff
@@ -45,6 +48,51 @@ class CommunityAPIReferralEventService(
 
         communityAPIClient.makeAsyncPostRequest(communityApiSentReferralPath, referRequest)
       }
+      ReferralEventType.CANCELLED,
```

Comment:
not a blocker but it would be nice to remove some of the duplicated code between these two branhes